README.md

```diff
@@ -1,28 +1,87 @@
+  external_id = "${var.cloudability_external_id}"
 }
 ```
 
+Add the External ID to  the tfvars file (config.auto.tfvars)
+```
+cloudability_external_id = "6124f6fd-6964-4fa6-9047-xxxxxxxxxxx"
+```
+
+_Without the external_id the role is not created_
+
+## Role already exists
+
+When it already exists terraform gives this error:
+```
+* aws_iam_role.cloudability_role: Error creating IAM Role CloudabilityRole: EntityAlreadyExists: Role with name CloudabilityRole already exists.
+	status code: 409, request id: xxxxxx-xxxx-xxxx-xxxx-xxxxxxxxx
+```
+
+### Import The role:
+
+To import the resource Terraform has the following command available:
+`terraform  import  module.cloudability.aws_iam_role.cloudability_role  CloudabilityRole`
+
+The wrapper script has this option to include the assume role and run import for this account
+`./runall.nl -b <accountid-description> -t module.cloudability.aws_iam_role.cloudability_role  -v CloudabilityRole`
+
+This will output the current policy. Copy the external id from the assume_role_policy and use this in the tfvars
+
+```
+id                    = CloudabilityRole
+arn                   = arn:aws:iam::xxxxxxx:role/CloudabilityRole
+assume_role_policy    = {"Version":"2012-10-17","Statement":[{"Effect":"Allow","Principal":{"AWS":"arn:aws:iam::xxxxxxxx:user/cloudability"},"Action":"sts:AssumeRole","Condition":{"StringEquals":{"sts:ExternalId":"xxxxxx-xxxx-xxxx-xxxx-xxxxxxxxx"}}}]}
+create_date           = 2017-11-16T10:58:38Z
+description           =
+force_detach_policies = false
+max_session_duration  = 3600
+name                  = CloudabilityRole
+path                  = /
+unique_id             = xxxxxx
+```
+
+```
+cloudability_external_id = "<UUID given by cloudability or dummy"
+```
+_there needs to be a string in the variable or the module isn't loaded_
+
+
+
+
 ## Inputs
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
-| external_id | The external identifier to use, given to you by Cloudability | string | - | yes |
-| role_name | The name of the role Cloudability will use | string | `CloudabilityRole` | no |
-| trusted_account_id | The Cloudability account this role will trust | string | `165736516723` | no |
+| automation_policy_name | Name of automation resource policy | string | `CloudabilityAutomationPolicy` | no |
+| enabled | Enable Policies and role | string | `true` | no |
+| external_id | The external identifier to use, given to you by Cloudability | string | `xxxxxx-xxxx-xxxx-xxxx-xxxxxxxxx` | no |
+| monitor_policy_name | Name of monitor resource policy | string | `CloudabilityMonitorResourcesPolicy` | no |
+| policy_path | Path in IAM for policies | string | `/` | no |
+| role_name | Name of Cloudability Role | string | `CloudabilityRole` | no |
+| role_path | Path in IAM for role | string | `/` | no |
+| trusted_account_id | Trusted Account from Cloudability | string | `xxxxxxxx` | no |
 
-## License
```

Comment:
As this is OS, the License should be specified

Reply:
I wrote the previous License. It was deleted by mistake